### ISSUE TITLE
[Task2]: Improve the MLP constructor in VISSL by replacing 2mlp constructor with 1mlp constructor

### ISSUE THREAD

**Issue by prigoyal** [MLH-Fellowship/vissl#5]
**Task**: Currently in VISSL, if we want to stack a specific model layer named "mlp", we have to specify it multiple times in the config file. For example: https://github.com/facebookresearch/vissl/blob/master/configs/config/pretrain/simclr/models/regnet400mf.yaml#L15-L16 
However, this is not very intuitive a user and user expectation would be that following should just work.
```
HEAD:
      PARAMS: [
        ["mlp", {"dims": [440, 440, 128], "use_relu": True}],
      ]
```

Our goal in this task is to enable the above. 

**Steps/Tips**:
1. In the `mlp.py` file https://github.com/facebookresearch/vissl/blob/master/vissl/models/heads/mlp.py#L54-L55 , we need to replace the line 54 by `for i, dim in enumerate(dims[1:])` and  insert a check after line 55 `if i == (len(dims) - 2): break`. Make these changes in the `mlp.py`
2. Next, make changes in the config files under that show up in search https://github.com/search?q=%22mlp%22+repo%3Afacebookresearch%2Fvissl+path%3A%2Fconfigs%2Fconfig%2Fpretrain&type=Code&ref=advsearch&l=&l= to combine the "mlp" from 2 times to 1 time (as described in the task description above). Exception is that: don't make changes to the files in folders https://github.com/facebookresearch/vissl/tree/master/configs/config/pretrain/pirl/models and https://github.com/facebookresearch/vissl/tree/master/configs/config/pretrain/jigsaw/permutations
3. Run all the CPU unit tests and make sure they pass
4. Send a PR to VISSL and tag Priya as reviewer

**Comment by prigoyal** [comment]
gist:

PARAMS: [
    ["mlp", {"dims": [440, 440], "use_relu": True}],
    ["mlp", {"dims": [440, 128]}]
  ]
  
#########################################
  PARAMS: [
    ["mlp", {"dims": [440, 440, 128], "use_relu": True}],
  ]
  
  PARAMS: [
    ["mlp", {"dims": [440, 440, 360, 128], "use_relu": True}],
  ]

  FC(440, 440) -> Relu -> FC(440, 128)
  
  FC(440, 440) -> Relu -> FC(440, 360) -> Relu -> FC (360, 128)
  
################################################################################
  for dim in dims[1:]:
    layers.append(nn.Linear(last_dim, dim, bias=use_bias))
    
  for i, dim in enumerate(dims[1:]):
      ..... line 55
      if i == (len(dims) - 2): 
         break
